python/delta_sharing/tests/test_rest_client.py

```diff
@@ -385,11 +387,95 @@ def test_list_files_in_table_partitioned_different_schemas(
     ]
 
 
+@pytest.mark.skipif(not ENABLE_INTEGRATION, reason=SKIP_MESSAGE)
 def test_list_table_changes(
     rest_client: DataSharingRestClient,
 ):
-    try:
-        rest_client.list_table_changes(Table(name="x", share="y", schema="z"), CdfOptions())
-        assert False
-    except Exception as e:
-        assert isinstance(e, HTTPError)
+    # The following table query will return all types of actions.
+    cdf_table = Table(name="cdf_table_with_partition", share="share1", schema="default")
+    response = rest_client.list_table_changes(
+        cdf_table,
+        CdfOptions(starting_version=0, ending_version=3)
```

Comment:
Better to add a test case with `starting_timestamp`, to just ensure it's correctly encoded(so that the server is able to parse it). 

But one thing to note is that, seems that when running tests locally the server timezone is PST, and when github runs tests the server timezone is GMT. So the same string `startingTimestamp` will be parsed as different timestamp (long), which will map to different version of the table, thus results in different testing behavior.

One trick I did is use a very large startingTimestamp and expect an error message: in https://github.com/delta-io/delta-sharing/pull/146 , DeltaSharingRestClientSuite:`getCDFFiles with Timestamp`,

Reply:
Added a test for start timestamp.

Reply:
As mentioned, needs to add a corresponding test case to test the encoding behavior is correct in our client, could you separate it out, and expect an error, similar to https://github.com/delta-io/delta-sharing/pull/146?

Though agree with you, we need to discuss whether it's safe to always expose `DeltaStandaloneException`, but in this case, we have to expose it because it's not internal error, and for the user/client to know that they need to set correct timestamp.

Reply:
Done.
Also added a more specific exception mapping in the reader, which will be safer.